docs/context.rst

```diff
@@ -385,12 +385,12 @@ following:
   See examples in :ref:`metadata-indexes`.
 - The entries in the Archive do not need to be per-detector.  You can
   specify results for a whole group of detectors, if that group is
-  enumerated in the DetDb.  For example, if DetDb contains a column
-  ``band``, the dataset could contain columns ``dets:band`` and
-  ``cal`` and simply report one calibration number for each frequency
-  band.  (On load, the Context Metadata system will automatically
-  broadcast the ``cal`` number so that it has shape ``(n_dets,)`` in
-  the fully populated AxisManager.)
+  enumerated in the DetDb (or added to det_info using other metadata).
+  For example, if DetDb contains a column ``band``, the dataset could
```

Comment:
Band is a highly overloaded term in SO world. Could we make a habit of using something like bandpass, fcode, freq_code, etc? I use entries with `bandpass = "f090"` in the detector wafer info.